### ISSUE TITLE
v2-beta.xbl.io Domain Unavailable

### ISSUE THREAD

**Issue by attheshow** [OpenXBL/OpenXBL-PHP#6]
The domain that's set as the $base_url in this repo doesn't appear to be active.

**Comment by attheshow** [comment]
It looks like this issue is fixed in dev-master but not in the stable release. Looks like a new release would be a good idea. 😊

**Comment by B1GSTACK** [comment]
Actually considering this was a the beta for v2.. and was long since merged to prod... alone with v3 thats merged into prod. Probably best to just remove this from existence :)

**Comment by dregimbal1** [comment]
This will be resolved once PR #7 is merged and tagged.
